packages/llm/router.py

```
@@ -0,0 +1,950 @@
+                    json={"model": spec.id, "input": [text for _i, text in pending]},
+                    headers={"Content-Type": "application/json",
+                             **provider.auth_headers(api_key)},
+                    timeout=60.0,
+                )
+                if response.status_code >= 400:
+                    raise TransientError(f"{spec.provider} HTTP {response.status_code}")
+                vectors = [
+                    list(entry.get("embedding") or [])
+                    for entry in (response.json().get("data") or [])
+                ]
+            except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)